its/src/org/sonarlint/eclipse/its/reddeer/views/SonarLintConsole.java

```diff
@@ -70,6 +70,16 @@ public void enableAnalysisLogs() {
     }
   }
 
+  public void enableIdeSpecificLogs(boolean enabled) {
+    consoleView.activate();
+    var menu = new ToolItemMenuItem(new DefaultToolItem(consoleView.getCTabItem().getFolder(), "Configure logs"), "IDE-specific traces");
+    if (!menu.isSelected() && enabled) {
```

Comment:
identical implementation in if-else branches??

Reply:
No, the `menu.select()` is both used for selecting and de-selecting :face_in_clouds: I can merge the branches tho and add a comment for when it applies:
- we don't want it enabled, but currently it is
- we want it enabled, and currently, it is not

Reply:
Done